src/Host/Host.php

```
@@ -235,7 +235,12 @@ public function addLabels(array $labels): self
 
     public function getLabels(): ?array
     {
-        return $this->config->get('labels', null);
+        $labels = $this->config->get('labels', null);
+        if (is_array($labels)) {
```

PHPStan (2.1.15)

Call to function is_array() with array will always evaluate to true.
Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your %configurationFile%.

(function.alreadyNarrowedType)